tasks/gene_typing/variant_detection/task_snippy_variants.wdl

```diff
@@ -111,6 +120,7 @@ task snippy_variants {
     String snippy_variants_ref_length = read_string("REFERENCE_LENGTH")
     String snippy_variants_ref_length_passed_depth = read_string("REFERENCE_LENGTH_PASSED_DEPTH")
     String snippy_variants_percent_ref_coverage = read_string("PERCENT_REF_COVERAGE")
+    String snippy_variants_percent_reads_aligned = read_string("PERCENT_READS_ALIGNED")
```

Comment:
Can you add this output to both workflows that use this task? I think it's just Snippy_Streamline and Snippy_Variants. Otherwise, the user won't be able to see this output in Terra.

Reply:
Yes it was on my list of things to do this week. thanks for the reminder.